src/osp/Resource/AssetImporter.cpp

```diff
@@ -151,7 +169,8 @@ void osp::AssetImporter::load_sturdy(TinyGltfImporter& gltfImporter, Package& pk
     {
         using Magnum::MeshPrimitive;
 
-        std::string const& meshName = gltfImporter.meshName(i);
+        std::string const& meshName = static_cast<std::string>(resPrefix)
+            + gltfImporter.meshName(i);
```

Comment:
This kinda sucks but I'm not sure how to avoid it, as the string needs to be modified. Open to suggestions.

Reply:
Definately cannot be casting from std::string_view to std::string. Big no-no. Call the std::string constructor directly.

Regardless, this is the wrong approach.

Use string_concat.

Copy it from here: https://github.com/splinter-build/splinter/blob/15-default-values/src/string_concat.h

Or wait for me to open a PR with a simplified version tomorrow.